setup.py

```diff
@@ -23,23 +23,20 @@
 with open(os.path.join(os.path.dirname(__file__), 'README.md'), 'r') as f:
     long_description = f.read()
 
-
-
-
 setup(
-    name='afs',
+    name='afs2-model',
     version=version,
     description='For AFS developer to develop analytics',
     long_description=long_description,
     author='benchuang',
```

Comment:
Use a team name or personal name?

Reply:
The supervisor said that `WISE-PaaS/AFS`, referenced AWS SDK.